data/sql/updates/pending_db_world/rev_1601314686326215700.sql

```diff
@@ -40,7 +40,7 @@ DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 11613);
 INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
 (11613, 0, 0, 0, 4, 0, 100, 0, 0, 0, 0, 0, 12, 11614, 1, 30000, 1, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Huntsman Radley - On Aggro - Summon Creature \'Bloodshot\''),
```

Comment:
@Knindzagxg  I noticed something interesting this time about this...
She already has one out at normal spawn... so when you get in combat with her, she will execute this and spawn the second one, is still ok that she has 2 of this minion at combat then (the original + the summon)?

The original and permanent one is:
NPC ID: 11614
GUID: 52640
Coords: X= 2328.97  Y= -2070.82  Z= 168.62
![image](https://user-images.githubusercontent.com/61223313/94999961-3f3c5680-057a-11eb-92ff-0a3608e939b3.png)

Reply:
Looks like Blizz decided to not use spell for spawn and have it up always.
Sorted it as well.